notebooks/book1/02/centralLimitDemo.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>Currently this function looks very ambigous, I had a discussion with zeel regarding this. we should change such that it can be interpretable easily. </p><p>a) <code>bins</code> argument is useless here b) N should be a number, making it as a array is confusing.  </p><p>We want to do following step:</p><ol><li>Take N samples from Beta and sum over it, it will give us single mean. make a function for the same.</li><li>Repeat step-1 sampleSize (10000) times using <code>jax.vmap</code>  and this will return means. make a function for the same. </li><li>Then plot histogram over means using <code>plot_convolutionHist()</code> but again take N as single number, not the array </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>